resources/image_metadata.json

```diff
@@ -891,7 +891,16 @@
   "tagReplace": {
     "^(tf)([0-9.]*)$": "TensorFlow $2",
     "^(pytorch)([0-9.]*)$": "PyTorch $2",
+    "^py(\\d)(\\d+)$": "Python $1.$2",
```

Comment:
Handling versions without dots using regular expressions isn't ideal since major versions can be two digits. It's better to use [explicit tag aliases](https://github.com/lablup/backend.ai-webui/blob/b0d7ea27566689e0babf18846ae7390829167cfe/resources/image_metadata.json#L822-L823) for such cases.

Reply:
But we have to update whenever python is updated. py312,py313,py314....